projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -27,6 +27,10 @@ public static function on_plugins_loaded() {
 			$publicize_ui = new Publicize_UI();
 
 		}
+		// Adding on a higher priority to make sure we're the first field registered.
+		// The priority parameter can be removed once we deprecate WPCOM_REST_API_V2_Post_Publicize_Connections_Field
+		add_action( 'rest_api_init', array( new Connections_Post_Field(), 'register_fields' ), 5 );
+
 		add_action( 'rest_api_init', array( new REST_Controller(), 'register_rest_routes' ) );
```

Comment:
Definitely not this PR, but I wonder if we should clean this up a bit. Feels odd to have a `REST_Controller` class in addition to other classes that add their own endpoints.

Reply:
Yeah, this is a rest field, so it's slightly different, but you're right, we could probably hook in a `rest_init` function that registered everything in one place, rather than two separate actions.